### DISCUSSION TITLE
Campaigns listing

### DISCUSSION THREAD

**Discussion by cesaromano** [amzn/ads-advanced-tools-docs#275] [Q&A]
There is a way to list campaigns including informations like "clicks" or "impressions"?

**Comment by kdwivedi13** [comment]
To get campaign performance data like clicks and impressions, you can kindly generate a campaign report for the respective ad product. For Sponsored Products campaigns, please use the "spCampaigns" report type. You may group this report by "campaign" and "adGroup", and include "impressions", "clicks", "cost" in the "columns" parameter to obtain those metrics. This [page](https://advertising.amazon.com/API/docs/en-us/guides/reporting/v3/report-types/campaign) provides more details and examples.

I hope this helps! Let me know if you have any other questions.
